src/server.js

```diff
@@ -0,0 +1,40 @@
+const FeedParser = require('feedparser')
+const request = require('request')
+const Rx = require('rx') 
+
+// Event loop will run 3 times for testing
+const interval = Rx.Observable.interval(5000).timeInterval().take(3)
```

Comment:
This introduces a very callback-y pattern and seems like it could get messy quickly. Is there another library that will give us promise support?